hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -2935,11 +2935,10 @@
     <description>
       The base dir for HTTP Jetty server to extract contents. If this property
       is not configured, by default, Jetty will create a directory inside the
-      directory named by the java.io.tmpdir System property（/tmp by default）.
```

Comment:
can you update the value of this property to `${ozone.metadata.dirs}/webserver`

Reply:
Thanks for the suggestion @jojochuang , I have changed it !!

Reply:
I think what @jojochuang meant is to increase the value of the property and not description.